<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>898</width>
    <height>604</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>481</width>
      <height>411</height>
     </rect>
    </property>
    <property name="title">
     <string>TreeView</string>
    </property>
    <widget class="QTreeView" name="treeView">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>461</width>
       <height>381</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>10</y>
      <width>381</width>
      <height>201</height>
     </rect>
    </property>
    <property name="title">
     <string>ListView</string>
    </property>
    <widget class="QListView" name="listView">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>20</y>
       <width>361</width>
       <height>171</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>220</y>
      <width>381</width>
      <height>201</height>
     </rect>
    </property>
    <property name="title">
     <string>TableView</string>
    </property>
    <widget class="QTableView" name="tableView">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>361</width>
       <height>171</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>430</y>
      <width>871</width>
      <height>121</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout" rowstretch="1,1">
     <item row="0" column="3">
      <widget class="QLabel" name="labFileName">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="5">
      <widget class="QLabel" name="labFileType">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QLabel" name="labSize">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="6">
      <widget class="QCheckBox" name="chkIsDir">
       <property name="text">
        <string>节点是目录</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3" colspan="3">
      <widget class="QLabel" name="labPath">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
